ukis_pysat/raster.py

```diff
@@ -85,7 +77,7 @@ def get_valid_data_bbox(self, nodata=0):
         :return: tuple with valid data bounds
         """
         valid_data_window = windows.get_data_window(self.__arr, nodata=nodata)
-        return windows.bounds(valid_data_window, windows.transform(valid_data_window, self.transform))
+        return windows.bounds(valid_data_window, windows.transform(valid_data_window, self.dataset.transform))
 
     def mask_image(self, bbox, crop=True, pad=False, mode="constant", constant_values=0):

```

Comment:
Since we did some api changes already, suggest to rename this method to "mask". "mask_image" is not consistent naming with "warp" and other methods in the "Image" class.

Reply:
I agree, but we will remove compatibility with 0.6.0